README.md

```diff
@@ -72,6 +72,30 @@ Log out from postgres and run:
 
 Your server is ready to run. You can do this by executing `rails server` and going to [http://localhost:3000](http://localhost:3000). Happy coding!
 
+## AngularJS Guide
+
+AngularJS is added inside `vendor/assets/javascripts` folder. The current version is `v1.6.3  `.
```

Comment:
I believe it's better not depending on a gem and just placing inside vendor the desired version of AngularJS.